add cloud9 instruction for update-behaviors-of-stack-resources (#216)
aws-samples/cfn101-workshop: content/intermediate/operations/update-behaviors-of-stack-resources/index.md

Reviewer: Missing cloud9 instructions for updating the stack for replacement of parameter

Reviewer: No need to implement this after confirming. Resolving.

Reviewer: Missing cloud9 instructions for updating the sack for "Updates with Some Interruption"

Reviewer: No need to implement this after confirming. Resolving.

Save your changes to the file. Next, create your stack with the `update-behaviors-of-stack-resources.yaml` template:

   :::::tabs{variant="container"}
	::::tab{id="cloud9" label="Cloud9"}
1. In the **Cloud9 terminal** navigate to `code/workspace/update-behaviors-of-stack-resources`:
    :::code{language=shell showLineNumbers=false showCopyAction=true}
    cd code/workspace/update-behaviors-of-stack-resources
    :::
    1. Use the AWS CLI to create the stack. The required parameters `--stack-name` and `--template-body` have been pre-filled for you.
    :::code{language=shell showLineNumbers=false showCopyAction=true}

>     aws cloudformation create-stack --stack-name update-behaviors-of-stack-resources --template-body file://update-behaviors-of-stack-resources.yaml

Reviewer: ```suggestion
    aws cloudformation create-stack --stack-name cfn-workshop-update-behaviors-of-stack-resources --template-body file://update-behaviors-of-stack-resources.yaml
```

    :::
    1. If the `create-stack` command was successfully sent, CloudFormation will return `StackId`.

>     :::code{language=shell showLineNumbers=false showCopyAction=true}

Reviewer: ```suggestion
    :::code{language=shell showLineNumbers=false showCopyAction=false}
```

>     "StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/drift-detection-workshop/739fafa0-e4d7-11ed-a000-12d9009553ff"

Reviewer: ```suggestion
    "StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/cfn-workshop-update-behaviors-of-stack-resources/739fafa0-e4d7-11ed-a000-12d9009553ff"
```

> 8. On the **Review** page, scroll down to the bottom, and choose **Create stack**.

Reviewer: ```suggestion
8. On the **Review** page, scroll down to the bottom, and choose **Submit**.
```
